sw/host/opentitanlib/src/ownership/owner.rs

```diff
@@ -219,12 +224,38 @@ impl OwnerBlock {
         })
     }
     pub fn sign(&mut self, key: &EcdsaPrivateKey) -> Result<()> {
+        ensure!(
```

Comment:
Is this particular check required in the boot services that were updated to support detached signatures?

Reply:
The client (opentitantool commands) will print a message when you send a `boot-svc` request that requires an attached signature.

The server (firmware) can support ECDSA signatures in either attached or detached form.